deploy/examples/create-external-cluster-resources.py

```diff
@@ -14,26 +14,26 @@
 limitations under the License.
 """
 
+import argparse
+import configparser
 import errno
-import sys
+import hmac
```

Comment:
are you using black python formatter?
This import should not move..

Reply:
The current import is not ordered, this is auto changed by my editor to keep it ordered. if we are planning to keep un ordered imports i can revert it back

Reply:
As the ci pass, the lint is fine, I am fine with this change